### PR TITLE
routing changes pages, simple base styling added

### PR DESCRIPTION
**Pull request by st0nebreaker**
# PR Description
Please include a summary of the change:
Simple base styling added to application. Routing rigged up for About, AllCocktails, and MyCocktail Components.

Commit Emojis:
:tada: Initial commit
:construction: HTML modifications/updates.
:art: CSS modifications/updates.
:crystal_ball: JS modifications/updates.
:recycle: refactor code.
:camera_with_flash: adding screenshots.
:bento: Adding new assets or images.
:beetle: If it is a bug or issue fix, describe the issue that was fixed.
:mute: remove logs and empty space/linter.
:bar_chart: new tests
:white_check_mark: add or update tests

If it is a new feature, describe the feature and the iteration of the project spec it addresses.

Note any questions or struggles as well.

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Tests (non-breaking change which adds testing)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/About/About.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```
Comment (Melizzo) [inline src/About/About.tsx:1]: Nice work on creating a basic About component.

File: src/AllCocktails/AllCocktails.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```
Comment (asodemann18) [inline src/AllCocktails/AllCocktails.tsx:1]: thanks for setting up this component!

File: src/App/App.scss [hunk 3]
```diff
@@ -13,6 +13,38 @@
   }
 }
 
+main {
+	background-image: url('https://images.unsplash.com/photo-1470337458703-46ad1756a187?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1949&q=80');
```
Comment (asodemann18) [inline src/App/App.scss:17]: Love the background image!

File: src/App/App.scss [hunk 4]
```diff
@@ -13,6 +13,38 @@
...
+	height: 100vh;
+}
+
+button {
+	background: none;
+	color: white;
+	border: 1px solid red;
+	height: 2vh;
+	border-radius: 2px;
+	&:hover {
```
Comment (asodemann18) [inline src/App/App.scss:34]: Nice job with the added styling on hover!

**Review by Melizzo — APPROVED** [review]

**Review by asodemann18 — COMMENTED** [review]
looks great!

**Review by asodemann18 — APPROVED** [review]
everything looks great!

**Pull request merged**